plugins/webp-uploads/settings.php

```diff
@@ -210,7 +210,19 @@ function webp_uploads_use_picture_element_callback(): void {
 	</div>
 	<div id="webp_uploads_picture_element_fieldset" class="<?php echo ! $jpeg_fallback_enabled ? 'disabled' : ''; ?>">
 		<label for="webp_uploads_use_picture_element" id="webp_uploads_use_picture_element_label">
-			<input name="webp_uploads_use_picture_element" type="checkbox" id="webp_uploads_use_picture_element" aria-describedby="webp_uploads_use_picture_element_description" value="1"<?php checked( webp_uploads_is_picture_element_enabled() ); ?> class="<?php echo ! $jpeg_fallback_enabled ? 'disabled' : ''; ?>" >
+			<input
+				type="checkbox"
+				id="webp_uploads_use_picture_element"
+				aria-describedby="webp_uploads_use_picture_element_description"
+				<?php checked( get_option( 'webp_uploads_use_picture_element', false ) ); // Option intentionally used instead of webp_uploads_is_picture_element_enabled() to persist when perflab_generate_webp_and_jpeg is updated. ?>
```

Comment:
Why not use `$picture_element_enabled` from above?
```suggestion
				<?php checked( $picture_element_enabled ); // Option intentionally used instead of webp_uploads_is_picture_element_enabled() to persist when perflab_generate_webp_and_jpeg is updated. ?>
```

Maybe rename the variable to something like `$picture_element_option` to clarify what the comment says?

Reply:
Yeah, the reason that `$picture_element_enabled` isn't used is due to the reason in the comment. We need the underlying DB value to be used, not the computed value as returned by `webp_uploads_is_picture_element_enabled()`. I've improved the variable naming in 7a50264e77f4f3050b3ccbed2f2149aa111a0437 and reused the variable here in 67485b434565f6d6be2e9a9b6a39bd57fa100a00.